conf/README.md

```diff
@@ -0,0 +1,123 @@
+# Config directory
+
+In this directory we store all the configuration for clusters and OCSCI config
+files.
+
+During the execution, via
+[ocsci pytest plugin](../pytest_customization/ocscilib.py), we are loading all
+the configs described below, combining them togehter and using
+[jinja2](http://jinja.pocoo.org/docs/2.10/templates/#variables)
+templates rendering variables which you can reuse among different configs from
+DEFAULT and different sections.
+
+This gives us the ability to easily reuse variables from context of the
+yaml file itself.
+
+Cause you can overwrite data just in specific section like for ENV_DATA
+(see below) and if you don't overwrite data in DEFAULT section, someone can
+still access old default value from DEFAULT section directly. See example of
+[this cluster config file](./examples/ocs_basic_install.yml) for more details.
+
+## OCS CI Config
+
+We moved all the OCSCI framework related config under
+[ocsci folder](./ocsci/). You can see we have defined here:
+
+* [default_config.yaml](ocsci/default_config.yaml) - this is the default
+    config file for OCS CI which is loaded automatically (don't pass it to
+    --ocsci-conf). It contains multiple sections which you can overwrite in
+    some custom config file you create or in feature we will have more config
+    files defined under `ocsci` folder as well which you can reuse.
+
+## Cluster config
+
+If you would like to overwrite cluster default data you can create file
+similar to [this example](./examples/ocs_basic_install.yml). You can see we
```

Comment:
s/You can see we can overwrite default data like/example shown overwrites below ENV data/